testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1515,6 +1505,10 @@
                    echo '#{server_node.public_ip} #{server_node.full_hostname} #{old_hostname}' >> /etc/hosts &&
                    echo '#{server_node.public_ip} #{new_hostname}#{server_node.full_hostname.delete_prefix(server_node.hostname)} #{new_hostname}' >> /etc/hosts")
   get_target('server', refresh: true) # This will refresh the attributes of this node
```

Comment:
In theory, Rubocop will raise if we write comments on the same line than our code.

Reply:
This was nothing I added, but I put the comment above the code.